pkg/network/setup/configstate.go

```diff
@@ -84,12 +91,20 @@ func (c ConfigState) Run(nics []podNIC, preRunFunc func([]podNIC) ([]podNIC, err
 		if preErr != nil {
 			return preErr
 		}
+		if c.podIfaceNameByNetwork == nil {
+			c.podIfaceNameByNetwork = map[string]string{}
+		}
+		for _, nic := range nics {
```

Comment:
I don't understand the suggestion. It will cause the map to never be initialized.
The initial value of `c.podIfaceNameByNetwork` is nil and not an empty map to differentiate between an empty map (pod with no interfaces) to a map that was never initialized.